MainUI.cs

```diff
@@ -60,16 +61,25 @@ private void LoadTransactionsOnUI(List<Transaction> list)
         {
             flowLayoutPanelTransactions.Controls.Clear();
             Console.WriteLine("search " + list.Count);
-            foreach (Transaction t in list)
+            if((list != null) && (list.First() != null))
             {
-                ListItemTransactions item = new ListItemTransactions
+                foreach (Transaction t in list)
```

Comment:
a bit cleaner than having continue inside the loop would be as follows:
` foreach (Transaction t in list.Where(item => item != null))`